Formula/osinfo-db.rb

```diff
@@ -0,0 +1,17 @@
+class OsinfoDb < Formula
+  desc "Osinfo database of information about operating systems for virtualization provisioning tools"
+  homepage "https://libosinfo.org"
+  url "https://releases.pagure.org/libosinfo/osinfo-db-20180514.tar.xz", :using => :nounzip
```

Comment:
This should be vendored into the other formula as a resource, it doesn't need to be separate.

Reply:
Alright, I will make it a resource of libosinfo, and use osinfo-db-tools as build dependency.
Then other packages that depend on libosinfo won't need to fix anything, e.g.: https://github.com/jeffreywildman/homebrew-virt-manager/issues/64